examples/draft-0-9-1/rich/rich.html

```diff
@@ -24,8 +24,8 @@
   </head>
   <body>
     <div id="target"></div>
-    <script src="../../../node_modules/react/dist/react.js"></script>
-    <script src="../../../node_modules/react-dom/dist/react-dom.js"></script>
+    <script src="../../../node_modules/react/umd/react.production.min.js"></script>
```

Comment:
can we update this PR to not use the production minified dependency ? It's useful to use the dev version for debugging purposes

Reply:
the `.development.js`  variants